utility.py

```diff
@@ -0,0 +1,14 @@
+import jwt
+secretkey='poojasecretkey'
+def encode(contact):
+    token = jwt.encode({'contact': contact}, secretkey)
+    print(token)
+    return token
+
+def decode(token):
+    dec = jwt.decode(token, secretkey, 'HS256')
+    print(dec)
+    return dec
+
```

Comment:
Add following line of code to line no 12,13,14:
if __name__=="__main__":
    token=jwtEncode(9088554022)
    jwtDecode(token)